src/bindings/jna/README.md

```diff
@@ -23,10 +23,18 @@ If this fails, make sure you have at least Java version 11 or higher installed a
 
 To use the bindings in a Java project, we have to include the jar file
 libelektra-\$VERSION.jar in the project. The version number is the same
-one as used for Elektra. This jar is created upon build of Elektra if
-you enable the JNA bindings, e.g., with `cmake -DBINDINGS=jna`, see also
-[COMPILE](/doc/COMPILE.md#bindings). Internally, Gradle will be used to
-actually compile the plugin.
+one as used for Elektra.
+
+### Build
+
+This jar is created upon building Elektra if you include the JNA bindings during the build process, e.g., with `cmake -DBINDINGS=jna`. Check out [COMPILE](/doc/COMPILE.md#bindings) for more details on how to include bindings in the build process.
+To make sure the build will be successfull, make sure the following requirements are met:
+- `gradle` must be installed on your machine, otherwise jna will not be included during compilation.
+- Use a JDK version <=17. Higher versions will result in *Unsupported class file major version 62/63* errors.
```

Comment:
## Review

### Documentation
Documentation is introductory, concise, good to read and describes everything what the PR does

### Examples
New/Changed documentation does not have to include example.

### Code is conforming
As restyled fails, it does not conform to the markdown guidelines.

```suggestion
- Use a JDK version <=17. Higher versions will result in _Unsupported class file major version 62/63_ errors.
```
This might resolve this according to restyled.

### APIs are conforming
This documentation does not include any new API or changes.

### Code is consistent
This PR does not include any new or changed source code.